orchagent/fdborch.cpp

```diff
@@ -171,7 +171,7 @@ void FdbOrch::update(sai_fdb_event_t type, const sai_fdb_entry_t* entry, sai_obj
 
             if (!m_portsOrch->getPort(entry->bv_id, vlan))
             {
-                SWSS_LOG_ERROR("FdbOrch notification: Failed to locate vlan\
+                SWSS_LOG_WARN("FdbOrch notification: Failed to locate vlan\
```

Comment:
Please change to NOTICE as other places have it as NOTICE for the same case. Also could you make the change to master to be consistent - https://github.com/Azure/sonic-swss/blob/master/orchagent/fdborch.cpp#L360

Reply:
@prsunny modified ERR log to NOTICE log
raised a PR for master https://github.com/Azure/sonic-swss/pull/1595